core/src/main/java/org/testcontainers/containers/Container.java

```diff
@@ -392,6 +393,17 @@ ExecResult execInContainer(String... command)
     ExecResult execInContainer(Charset outputCharset, String... command)
                     throws UnsupportedOperationException, IOException, InterruptedException;
 
+    /**
+     *
+     * Method allow to copy file which we have in our repository to docker container
```

Comment:
I also think it's a file inside the classpath and not inside the repository, correct?